content/docs/forms.md

```diff
@@ -178,33 +178,33 @@ class FlavorForm extends React.Component {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/JbbEzX?editors=0010)
+[**इसे CodePen पर आज़माएँ**](https://codepen.io/gaearon/pen/JbbEzX?editors=0010)
 
-Overall, this makes it so that `<input type="text">`, `<textarea>`, and `<select>` all work very similarly - they all accept a `value` attribute that you can use to implement a controlled component.
+सब मिलाकर, अर्थ यह है की `<input type="text">`, `<textarea>`, और `<select>` सब एक जैसे ही काम करते हैं - यह सब एक `value` एट्रिब्यूट एक्सेप्ट करते हैं जिसे आप एक कंट्रोल्ड कौम्पोनॅन्ट इम्प्लीमेंट करने के लिए इस्तेमाल कर सकते हैं।
 
-> Note
+> ध्यान दीजिये
 >
-> You can pass an array into the `value` attribute, allowing you to select multiple options in a `select` tag:
+> आप एक array को `value` एट्रिब्यूट में भेज सकते हैं, जिससे आप `select` टैग से बहुत ऑप्शंस का चयन कर सकते हैं:
 >
 >```js
 ><select multiple={true} value={['B', 'C']}>
 >```
 
-## The file input Tag {#the-file-input-tag}
+## फाइल इनपुट टैग {#the-file-input-tag}
 
-In HTML, an `<input type="file">` lets the user choose one or more files from their device storage to be uploaded to a server or manipulated by JavaScript via the [File API](https://developer.mozilla.org/en-US/docs/Web/API/File/Using_files_from_web_applications).
+HTML में, एक `<input type="file">` यूजर को चयन करने देता है, अपने डिवाइस की स्टोरेज से एक या अनेक file जिसे या तो सर्वर पर अपलोड किया जाता है या जावास्क्रिप्ट के [File API](https://developer.mozilla.org/en-US/docs/Web/API/File/Using_files_from_web_applications) से मैनिपुलेट किया जाता है।
 
 ```html
 <input type="file" />
 ```
 
-Because its value is read-only, it is an **uncontrolled** component in React. It is discussed together with other uncontrolled components [later in the documentation](/docs/uncontrolled-components.html#the-file-input-tag).
+क्यूंकि इसकी वैल्यू read-only होती है, यह React में एक **uncontrolled** कौम्पोनॅन्ट है। इसके बारे में विस्तार से दूसरे अनकंट्रोल्ड कौम्पोनॅन्टस [डॉक्यूमेंटेशन में बाद में](/docs/uncontrolled-components.html#the-file-input-tag) के साथ चर्चा की गई है।
```

Comment:
`दूसरे अनकंट्रोल्ड कौम्पोनॅन्टस [डॉक्यूमेंटेशन में बाद में](/docs/uncontrolled-components.html#the-file-input-tag) के साथ चर्चा की गई है।` => `दूसरे अनकंट्रोल्ड कौम्पोनॅन्टस के साथ [डॉक्यूमेंटेशन में बाद में](/docs/uncontrolled-components.html#the-file-input-tag) चर्चा की गई है।`